Makefile

```diff
@@ -48,7 +48,7 @@ release: build-release
 
 release-debug: build
 	godot --export-debug "Android" ./native/bin/android/{{project-name}}.debug.apk
-	godot --export-debug "Linux/X11" ./target/{{project-name}}.debug.x86_64
+	godot --export-debug "Linux/X11" ./native/bin/linux-x11/{{project-name}}.debug.x86_64
```

Comment:
I'm not sure if you'd actually want to export into the project folder itself, alongside GDNative binaries that are supposed to be assets...? But I'm also not sure how this can be improved with the current directory structure. Maybe an separate `export` directory under the project root?

As an aside, the user guide currently [suggests](https://godot-rust.github.io/book/getting-started/hello-world.html#creating-the-project) that the Rust crate be put in separate directories from the Godot project, so we can be sure that there will be zero problems with the resource importer or exporting process. However, that's also up to discussion since there is also `.gdignore` that this repo makes use of.

I googled a bit but found no information on how (or whether) `.gdignore` works in the *export* process. This is important because I'm assuming that users won't want their Rust sources to be exported by default. Is there any information on this?

Reply:
I am trying to adapt the project utils and having the binaries in a directory other than `target/` complicates things too.

This is only tangentially related, but I do feel like the separation of the binary artefacts, be it exported games or shared libraries, could be better.

Reply:
We can add a filter on which resources are included/excluded in the exported game using this window (which can be configured via the `export_presets.cfg` file)
![export-window](https://user-images.githubusercontent.com/5382992/94124517-af1a5500-fe87-11ea-9cee-3201b2906458.png)

Reply:
`.gdignore` works only in the godot editor, it excludes things from showing on the editor's file manager

Reply:
Even though we could exclude the release binaries from export, I'm still not sure if they should be put alongside the shared libraries and resource files. It just feels wrong. I'll wait and see what karroffel has to say about this.

Reply:
I personally prefer to keep the Rust project and Godot project separate too, including binary files, so I'm with toasteater here :+1:

Reply:
> I personally prefer to keep the Rust project and Godot project separate too

Sure :+1: I will create a separate issue for separating the Rust project from the Godot Project. Let's merge this first